update: kotlinx-browser library (#5706)
JetBrains/kotlin-web-site: docs/topics/js/browser-api-dom.md

[//]: # (title: Browser and DOM API)

The [`kotlinx-browser`](https://github.com/Kotlin/kotlinx-browser) library lets you access browser-specific functionality.
It includes typical top-level objects such as `document` and `window` and provides typesafe wrappers

> for the functionality exposed by these objects where possible.

Reviewer: I think that "where possible" could be mentioned earlier...
Smth like 

> It includes common top-level objects such as `document` and `window` and, where possible, provides type-safe wrappers for their functionality.

WDYT?

> As a fallback, the `dynamic` type is used to provide interaction with functions that do not map well into the Kotlin type system.

Reviewer: ```suggestion

As a fallback, the `dynamic` type provides access to functions that do not map well into the Kotlin type system.
```

Reviewer: Does it work?

To use the browser and DOM API, add the `kotlinx-browser` library as a dependency to your project's build file:

> ```kotlin

Reviewer: Don't we want to add tabs or, at least, the name of the file as a comment?

//build.gradle.kts

repositories {
    mavenCentral()
}

kotlin {
    js {
        browser()
    }

    sourceSets {
        commonMain {
            dependencies {

>                 implementation("org.jetbrains.kotlinx:kotlinx-browser:0.5.0")

Reviewer: Don't we want to make it a var in the v.list? 🤔
